Ray.jl/src/runtime.jl

```diff
@@ -1,3 +1,28 @@
+function init()
+    # XXX: this is at best EXREMELY IMPERFECT check.  we should do something
+    # more like what hte python Worker class does, getting node ID at
+    # initialization and using that as a proxy for whether it's connected or not
+    #
+    # https://github.com/beacon-biosignals/ray/blob/dfk%2Fusing-Ray/python/ray/_private/worker.py#L421
```

Comment:
this gives me a 404 - I think you linked to you own branch?

Reply:
changed all these to use the git commit hash instead of branch name (and changed my editor config to generate those from now on 😁 )